ligo/types.mligo

```diff
@@ -37,7 +37,7 @@ type tick_state = {
         assuming that now i_c is also below (above) this tick.
 
         Here we assume that, during all the time since Unix epoch start till
-        the moment of tick initialization, i_c was set to -∞.
+        the moment of tick initialization, i_c was set to ∞.
```

Comment:
Why was this changed? :o the way it was before seemed correct to me...

Reply:
I'm looking in the article, and it says that the formula for `seconds_outside` initial value is

```
# Formula (6.25)
if i_c >= i 
  then Tezos.now - (0 : timestamp) 
  else 0  
```

The same does the current implementation.

I.e. if at the moment of initialization `i_c` is below us, we set `seconds_outside` to zero. So before the moment of initialization `i_c` can be treated to be always above our tick.

That's weird because the sentence right before the formula - "By convention, it is initialized as if every second since the Unix timestamp 0 had been spent below that tick" - seems to say the opposite. I may be missing something, maybe you can explain this to me.

Reply:
Ah wait, after reading your definition I see why it work this way, now it makes perfect sense.